core_data_modules/traced_data/traced_data.py

```diff
@@ -18,8 +18,8 @@ def __init__(self, user, source, timestamp):
         :type user: str
         :param source: Identifier of the program, or similar, which generated the new data.
         :type source: str
-        :param timestamp: When the updated data was generated.
-        :type timestamp: float
+        :param timestamp: When the updated data was generated, in timezone-aware ISO 8601 format.
```

Comment:
Could you stick an example string in here?

Reply:
added